src/jquery.waitforimages.js

```diff
@@ -209,6 +203,9 @@
 
                 });
 
+                if (typeof img.srcset !== "undefined") {
```

Comment:
Why is it safer?

If the property doesnt exist, it will be `undefined`, which is falsey.